modules/kubectl-wrapper/main.tf

```diff
@@ -15,7 +15,7 @@
  */
 
 locals {
-  base_cmd = "${var.cluster_name} ${var.cluster_location} ${var.project_id} ${var.internal_ip} ${var.use_existing_context}"
+  base_cmd = "${var.cluster_name} ${var.cluster_location} ${var.project_id} ${var.internal_ip} ${var.use_existing_context} ${var.impersonate_service_account}"
```

Comment:
@yashbhutwala changing the base command can result in a delete recreate, so lets add this as optional following this pattern 
https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/864#discussion_r611761537

Reply:
Thanks for the quick review @bharathkkb!  I made the change 😃.